### PR TITLE
Support imagees in the accessory view of a table cell

### PR DESCRIPTION
**Pull request by markrickert**
With test.

### CODE REVIEW

**Comment by markrickert** [comment]
Hmm.... looks like the travis failure is unrelated to my branch.

**Comment by bmichotte** [comment]
Ok, I found where is the crash ;)

```
# test_form_screen.rb
{
  title: 'Custom cell',
  name: :custom_cell,
  cell_class: MyCustomCell,
  value: 'Hello',

  # PM::TableViewCell stuff
  height: 86,
  selection_style: :none,
  image: {
       image: 'rubymotion-logo.png',
       size: 30
  }
}
```

So it tries to `UIImage.imageNamed(a_Hash)` :-)

File: lib/ProMotion/XLForm/xl_form_cell_builder.rb [hunk 1]
```diff
@@ -68,6 +68,12 @@ def create_cell(cell_data)
         cell_class = XLFormColorSelectorCell if cell_class.nil?
       end
 
+      # image accessory
+      if cell_data[:image]
```
Comment (bmichotte) [inline lib/ProMotion/XLForm/xl_form_cell_builder.rb:72]: `if cell_data[:image] && !cell_data[:image].is_a?(Hash)`

To correct crash ?

**Comment by markrickert** [comment]
Sorry - forgot about this one... did you fix the crash?

**Comment by bmichotte** [comment]
No problem, yes, with a check if it's a `Hash` or not

**Pull request merged**